implementations/rust/ockam/ockam_vault_core/Cargo.toml

```diff
@@ -23,4 +23,5 @@ heapless = { version = "0.7", features = ["serde"], optional = true }
 serde = {version = "1.0", features = ["derive"]}
 serde-big-array = "0.3"
 zeroize = { version = "1.1", features = ["zeroize_derive"] }
+either = "1.6.1"
```

Comment:
Is it used?

Reply:
It is not. Thanks for pointing this out. My first idea was to user Either type to store two types of signature arrays - one 64 bit long and the second one for BBS+